frontend/src/app/modules/interventions/details/delete.dialog.ts

```diff
@@ -3,14 +3,16 @@ import { Component } from '@angular/core';
 @Component({
     selector: 'app-delete-intervention-dialog',
     template: `
-        <h2 mat-dialog-title>Usuń zgłoszenie</h2>
+        <h2 mat-dialog-title>{{title}}</h2>
         <mat-dialog-content class="mat-typography">
             Czy jesteś pewien, że chcesz usunąć to zgłoszenie?
         </mat-dialog-content>
         <mat-dialog-actions align="end">
-            <button mat-button mat-dialog-close cdkFocusInitial>Nie</button>
-            <button mat-button [mat-dialog-close]="true">Tak</button>
+            <button mat-button mat-dialog-close cdkFocusInitial [attr.data-dialog-title]="title">Nie</button>
```

Comment:
should: Maybe use [withGtmContextData] directive, so that would be obvious in the future what the purpose of that additional attribute is?

Reply:
Done